src/AlignLoads.h

```diff
@@ -0,0 +1,19 @@
+#ifndef HALIDE_ALIGN_LOADS_H
+#define HALIDE_ALIGN_LOADS_H
+
+/** \file
+ * Defines a lowering pass the breaks unaligned loads into two aligned
+ * loads.
+ */
+#include "IR.h"
+#include "Target.h"
+namespace Halide {
+namespace Internal {
+
+Stmt align_loads(Stmt s, const Target &t);
```

Comment:
I think now that we are just calling it for Hexagon code, I think this should just take an alignment (instead of a target). That way there is no expectation that align_loads might do any of the logic that we were previously considering implementing (examining loop device APIs, etc.).